packages/system-tests/scenarios/scaffolding.test.ts

```diff
@@ -257,6 +257,33 @@ describe('Scaffolding commands', () => {
       await common.closeTab();
     }
   });
+
+  it('Should create Lightning LWC', async () => {
+    // Invoke SFDX: Create LWC Bundle command by name
+    await app.command('workbench.action.quickOpen');
+    await common.type('>SFDX: Create LWC Bundle');
+    await app.client.keys(['NULL', 'Enter', 'NULL'], false);
+    await app.wait();
+
+    const fileName = `lwc_${new Date().getTime()}`;
+
+    // Enter file name
+    await common.type(fileName);
+    await app.client.keys(['NULL', 'Enter', 'NULL'], false);
+    await app.wait();
+
+    // Enter desired location (without slashes so it's OS-independent)
+    await common.type('defaultlightningcomponents');
```

Comment:
I need to use the shortened `defaultlightningcomponents` unlike the full `force-appmaindefaultlightningcomponents` like we do with other tests here. If I use the full path, it doesn't actually find it.

I think it's because there are too many characters and it doesn't like it.